pkg/apis/app/v1beta1/kogitoinfra_types.go

```diff
@@ -60,28 +60,33 @@ type KogitoInfraSpec struct {
 	InfraProperties map[string]string `json:"infraProperties,omitempty"`
 }
 
+// RuntimeVariables defines the variables that will be
+// extracted from the linked resource and added to the
+// deployed Kogito service.
+type RuntimeVariables struct {
```

Comment:
wouldn't make sense rename it to something like RuntimeProperties?
envs are translated to app properties anyways.